src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -790,34 +793,25 @@ private boolean processInstance(File instanceDir) {
     EventBus.publish(new ExportProgressPercentageEvent((processedInstances * 100.0) / totalInstances, briefcaseLfd));
 
     // If we are encrypted, be sure the temporary directory
-    // that will hold the unencrypted files is created and empty.
+    // that will hold the unencrypted files is created.
     // If we aren't encrypted, the temporary directory
     // is the same as the instance directory.
 
     File unEncryptedDir;
     if (briefcaseLfd.isFileEncryptedForm()) {
-      // create or clean-up the temp directory that will hold the unencrypted
+      // create the temp directory that will hold the unencrypted
       // files. Do this in the outputDir so that the briefcase storage location
       // can be a read-only network mount. issue 676.
-      unEncryptedDir = new File(outputDir, ".temp");
-
-      if (unEncryptedDir.exists()) {
-        // silently delete it...
-        try {
-          FileUtils.deleteDirectory(unEncryptedDir);
-        } catch (IOException e) {
-          String msg = "Unable to delete stale temp directory: " + unEncryptedDir.getAbsolutePath();
-          log.warn(msg, e);
-          EventBus.publish(new ExportProgressEvent(msg, briefcaseLfd));
-          return false;
-        }
-      }
-
-      if (!unEncryptedDir.mkdirs()) {
+      unEncryptedDir = null;
+      try {
+        Path path = Files.createTempDirectory(Paths.get(outputDir.getAbsolutePath()), ".temp");
```

Comment:
We are refactoring everything we can to Java NIO2 artifacts... let's avoid using strings for everything and change `outputDir.getAbsolutePath()` to `outputDir.toURI()`. (Look those uppercase chars, @dcbriccetti! :open_mouth:)

Also, you're using `".temp"` as prefix, but that looks more like a suffix. I think that `"decryption"` would work there.

Reply:
I am using `".temp"` to hide it since earlier also the directory was `".temp"`. Should I refactor it to `"decryption"` or `".decryption"`

Reply:
Oh, ok! Then, leave it like that :)

Although that won't work on Windows...